src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -111,7 +119,7 @@ public class DriveTrain extends SubsystemBase {
           // heading:          0.001 rad
           // l and r velocity: 0.1   m/s
           // l and r position: 0.005 m
-          VecBuilder.fill(0.001, 0.001, 0.001, 0.1, 0.1, 0.005, 0.005));
+          VecBuilder.fill(0.000, 0.000, 0.000, 0.0, 0.0, 0.000, 0.000));
```

Comment:
Just curious, what do these numbers mean? why did you set them to 0

Reply:
These configure the simulation's error injection.  I set them to zero during debug to reduce variables, but I'll reset these back to defaults now that we think the code is working.